code/game/objects/items/control_wand.dm

```diff
@@ -29,6 +29,28 @@
 	to_chat(user, "Now in mode: [mode].")
 
 /obj/item/weapon/door_remote/afterattack(obj/machinery/door/airlock/D, mob/user)
+	var/turf/doorlocation = get_turf(D)
+	if(istype(D)) //yes, the istype has to be here, otherwise it runtimes if you misclick
+		if(D.is_special)
+			if(D.check_access(src.ID))
+				if(istype(D, /obj/machinery/door/airlock/hatch/gamma) && !(seclevel2num(get_security_level()) == SEC_LEVEL_GAMMA))
```

Comment:
Yes, but what about all of the different conditions the doors would need to
open in? They would still need to be there. I could have made it so that
they just cannot be opened at the start, but that was not my intention.